cmd/batch-job-common-types.go

```diff
@@ -288,3 +289,12 @@ func (s *BatchJobSize) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	*s = BatchJobSize(sz)
 	return nil
 }
+
+// getBatchJobIDSeparator - returns the separator to be used in the batch job ID
+// windows requires `_` as the separator `:` will be an invalid one
+func getBatchJobIDSeparator() string {
+	if runtime.GOOS == "windows" {
```

Comment:
```suggestion
	if runtime.GOOS == globalWindowsOSName {
```

Minor strictly not needed, and please fix

```diff
diff --git a/.github/workflows/vulncheck.yml b/.github/workflows/vulncheck.yml
index d12a20046f..4452575055 100644
--- a/.github/workflows/vulncheck.yml
+++ b/.github/workflows/vulncheck.yml
@@ -21,7 +21,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.22.3
+          go-version: 1.22.4
           check-latest: true
       - name: Get official govulncheck
         run: go install golang.org/x/vuln/cmd/govulncheck@latest
```

Reply:
`1.22.x` - there is no point in bumping every minor version manually

Reply:
I explained why it was because of what @aead suggested the correct way to run `vulncheck` that exposes we must to Go upgrade for our releases and make a new release.

Reply:
@harshavardhana @aead IMO we should run this on releases. Running it here is pointless, since we only need to check if the *release* is with the correct Go version - and this is not guaranteed to be the release version.